examples/pytorch/vision/Face_Detection/README_M4.md

```diff
@@ -0,0 +1,130 @@
+# Code for Face Detection Experiments with RNNPool
+## Requirements
+1. Follow instructions to install EdgeML operators and their pre-requisites [here](https://github.com/microsoft/EdgeML/blob/master/pytorch/README.md).
+2. Install requirements for face detection model using
+``` pip install -r requirements.txt ``` 
+We have tested the installation and the code on Ubuntu 18.04 with Python 3.6, Cuda 10.2 and CuDNN 7.6
+
+## Dataset - WIDER Face
+1. Download WIDER face dataset images and annotations from http://shuoyang1213.me/WIDERFACE/ and place them all in a folder with name 'WIDER_FACE'. That is, download WIDER_train.zip, WIDER_test.zip, WIDER_val.zip, wider_face_split.zip and place it in WIDER_FACE folder, and unzip files using: 
+
+```shell
+cd WIDER_FACE
+unzip WIDER_train.zip
+unzip WIDER_test.zip
+unzip WIDER_val.zip
+unzip wider_face_split.zip
+cd ..
+
+```
+
+2. In `data/config_qvga.py` , set _C.HOME to the parent directory of the above folder, and set the _C.FACE.WIDER_DIR to the folder path. 
+That is, if the WIDER_FACE folder is created in /mnt folder, then _C.HOME='/mnt'
+_C.FACE.WIDER_DIR='/mnt/WIDER_FACE'.
+
+3. Run
+``` python prepare_wider_data.py ```
```

Comment:
Failed with the following error:
(py37) harshasi@GPUnode1:~/EdgeML/examples/pytorch/vision/Face_Detection$ python3 prepare_wider_data.py
Traceback (most recent call last):
  File "prepare_wider_data.py", line 10, in <module>
    from data.config import cfg
  File "/home/harshasi/EdgeML/examples/pytorch/vision/Face_Detection/data/__init__.py", line 4, in <module>
    from .widerface import WIDERDetection
  File "/home/harshasi/EdgeML/examples/pytorch/vision/Face_Detection/data/widerface.py", line 10, in <module>
    from utils.augmentations import preprocess, preprocess_qvga
  File "/home/harshasi/EdgeML/examples/pytorch/vision/Face_Detection/utils/__init__.py", line 4, in <module>
    from .augmentations import *
  File "/home/harshasi/EdgeML/examples/pytorch/vision/Face_Detection/utils/augmentations.py", line 19, in <module>
    from data.choose_config import cfg
  File "/home/harshasi/EdgeML/examples/pytorch/vision/Face_Detection/data/choose_config.py", line 7, in <module>
    IS_QVGA_MONO = os.environ['IS_QVGA_MONO']
  File "/home/harshasi/anaconda3/envs/py37/lib/python3.7/os.py", line 678, in __getitem__
    raise KeyError(key) from None
KeyError: 'IS_QVGA_MONO'

Reply:
have updated readme to avoid error

Reply:
This should ideally be a different flag right? Because this needs to be run even when the images aren't monochrome?